pkg/provisioner/ironic/ironic.go

```diff
@@ -1772,6 +1793,11 @@ func (p *ironicProvisioner) hardPowerOff() (result provisioner.Result, err error
 	}
 
 	if ironicNode.PowerState != powerOff {
+		if ironicNode.LastError != "" {
```

Comment:
This is where we really need the force flag.

Reply:
Fixed it.